lib/TAP/Formatter/Session/TeamCity.pm

```diff
@@ -124,18 +124,25 @@ sub _handle_comment {
     }
     $comment =~ s/^\s*#\s?//;
     $comment =~ s/\s+$//;
-    return if $comment =~ /^\s*$/;
-    $self->_tc_test_output_buffer(
-        $self->_tc_test_output_buffer . "$comment\n" );
-    $self->_maybe_print_raw( $result->raw );
+    return unless $comment =~ /\S/;
+    $self->_append_to_tc_test_output_buffer("$comment\n");
 }
 
 ## no critic (Subroutines::ProhibitUnusedPrivateSubroutines, Subroutines::ProhibitExcessComplexity)
+#
+# This method will be called for all subtest output. TAP cannot parse subtests
```

Comment:
This could be better worded;  TAP's a spec, it doesn't parse.  The official TAP spec doesn't understand the convention used for subtests?  The TAP module we're talking about doesn't parse subtests?